types/rc-tree/index.d.ts

```diff
@@ -258,6 +264,10 @@ export interface TreeProps {
      * it execs when fire the tree's dragend event
      */
     onDragEnd?: (props: OnDragEndData) => void;
+    /**
+     * the data to be rendered
+     */
+    treeData?: [];
```

Comment:
This doesn't make any sense - it can only take an empty array? Did you mean `any[]` or `unknown[]` ? It'd be good to add a test to demonstrate what you want this to do.